lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -32,6 +32,8 @@ watchdog = "*"
 numpy = "*"
 bx-python = "*"
 MarkupSafe = "*"
+Markdown = "*"
+Weasyprint = "==0.42"
```

Comment:
@jmchilton Are these 2 new dependencies actually used anywhere? Weasyprint adds other 7 dependencies to our `requirements.txt`, see https://github.com/galaxyproject/galaxy/pull/8843/files

Reply:
They are not - can we remove them in #8843 or should I open a new PR?

Reply:
Since the dependencies rebuild wasn't run, they were never added to requirements.txt and I think it's probably safe to just drop them in #8843?

Reply:
Yep, I can do that.